### ISSUE TITLE
Would botocore be too much of an dependency?

### ISSUE THREAD

**Issue by kerma** [notifiers/notifiers#287]
**Is your feature request related to a problem? Please describe.**
I'd like to use notifiers with AWS services like SNS, SES and maybe SQS.

**Describe the solution you'd like**
For this to happen notifiers would probably need add an additional dependency: botocore.

**Describe alternatives you've considered**
Not use notifiers and implement the same thing in my app directly. 

**Additional context**
I'm thinking about writing an SNS provider. Need to know up front whether botocore would be considered too big of an dependency.

**Comment by liiight** [comment]
I actually considered adding extra dependencies at some point, exactly for this purpose. So you could essentially do `pip install notifiers[extra]` and have additional deps installed if you wish to do so. 

This would require some minimal changes to the framework and ofcourse implementing these new notifiers but it's very doable.

I'm changing this issue to a regular feature request. 

Thanks for taking an interest in notifiers!

**Comment by Coinhexa** [comment]
still no updates after 4 years?

**Comment by liiight** [comment]
yep, still nothing. It's almost like the passage of time itself is not a source of work.

**Comment by Coinhexa** [comment]
[Is it possible for me to somehow use this to send emails via SES?](https://notifiers.readthedocs.io/en/latest/providers/email.html)
